core/domain/question_services.py

```diff
@@ -56,8 +58,13 @@ def _migrate_state_schema(versioned_question_state):
             feconf.CURRENT_STATE_SCHEMA_VERSION)
 
     while state_schema_version < feconf.CURRENT_STATE_SCHEMA_VERSION:
-        question_domain.Question.update_state_from_model(
-            versioned_question_state, state_schema_version)
+        if state_schema_version == 29:
+            question_domain.Question.update_state_from_model(
+                versioned_question_state, state_schema_version,
+                question_model=question_model)
+        else:
+            question_domain.Question.update_state_from_model(
+                versioned_question_state, state_schema_version)
```

Comment:
Pass None for the question_model here instead of default params

Reply:
Done.